Documentation/compilation.rst

```diff
@@ -52,78 +81,88 @@ Installing HDF5
 
 .. code-block:: bash
 
-  wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.11/src/hdf5-1.8.11.tar.bz2
-  tar -xaf hdf5-1.8.11.tar.bz2
-  cd hdf5-1.8.11
-  CC=mpiicc FC=mpiifort ./configure --enable-parallel --prefix=$HOME --with-zlib --disable-shared --enable-fortran 
-  #NOTE: This is with Intel MPI, if you have to use a different MPI, please adjust mpiicc and mpiifort accordingly to the scripts that call icc and ifort!
-  #MPICH note: CC=mpicc FC=mpif90 ./configure --enable-parallel --prefix=$HOME --with-zlib --disable-shared --enable-fortran
-  make
+  git clone https://github.com/mortenpi/hdf5.git
+  cd hdf5
+  git checkout 8a275ab7831002f3e  # take a stable version (optional)
```

Comment:
Why the change to github repository? As far as i know, the previously used archive was a stable release